pkg/daemon/server/daemon_server.go

```diff
@@ -305,6 +305,24 @@ func (ds *daemonServer) exposeMetrics(ctx context.Context) {
 			} else {
 				watermarkCmpNow.WithLabelValues(ds.pipeline.Name).Set(float64(time.Now().UnixMilli() - maxWM))
 			}
+
+			pipelineStatus, err := ds.metaDataQuery.GetPipelineStatus(ctx, &daemon.GetPipelineStatusRequest{Pipeline: ds.pipeline.Name})
```

Comment:
`pipelineStatus` -> `pipelineDataHealth`

`pipelineStatus.Status.Status` -> this has a lot of nested status otherwise, 
`pipelineDataHealth.Status.Status` looks better